test/test_jit.py

```diff
@@ -7826,6 +7826,7 @@ def test_passing_too_few_args(self):
         ):
             torch.ops.aten.relu()
 
+    @unittest.skipIf(True, "temporarily disabled")
```

Comment:
The way you can fix this test is by changing `log_softmax` to some other ATen function that takes at least two positional parameters and doesn't have overloads. For example `transpose`. I'll have a more robust fix soon, but you can do this for now,